lib/cartopy/tests/mpl/conftest.py

```diff
@@ -19,3 +19,20 @@ def mpl_test_cleanup(request):
     finally:
         # Closes all open figures and switches backend back to original
         plt.switch_backend(orig_backend)
+
+
+def pytest_itemcollected(item):
+    mpl = item.get_closest_marker('mpl_image_compare')
```

Comment:
Perhaps a different name than `mpl`? This confused me thinking it was `import matplotlib as mpl` down below.